flink/README_unified_batch_streaming.md

```diff
@@ -95,30 +96,97 @@ UPDATE `test`.`test_cdc` SET id = 1 WHERE id = 2;
 
 You will find that the data in Flink is the same as the real data of tidb and is updated in real time.
 
-## 6 Configuration
+## 6 Delete in Streaming Mode
+
+Create source table and target table with same schema in TiDB:
+
+```sql
+CREATE TABLE `test`.`src_table`(
+    id BIGINT(20) PRIMARY KEY ,
+    name VARCHAR(16) ,
+    PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
+);
+
+CREATE TABLE `test`.`target_table`(
+    id BIGINT(20) PRIMARY KEY ,
+    name VARCHAR(16) ,
+    PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */
+);
+```
+
+Start Flink SQL client in streaming mode, create TiDB Catalog and insert:
+
+```sql
+SET 'sql-client.execution.result-mode' = 'table';
+
+CREATE CATALOG `tidb`
+WITH (
+  'type' = 'tidb',
+  'tidb.database.url' = 'jdbc:mysql://localhost:4000/',
+  'tidb.username' = 'root',
+  'tidb.password' = '',
+  'tidb.streaming.source' = 'kafka',
+  'tidb.streaming.codec' = 'json',
+  'tidb.streaming.kafka.bootstrap.servers' = 'localhost:9092',
+  'tidb.streaming.kafka.topic' = 'test_cdc',
+  'tidb.streaming.kafka.group.id' = 'test_cdc_group',
+  'tidb.streaming.ignore-parse-errors' = 'true',
+  'tidb.sink.impl' = 'TIKV',
+  'tidb.write_mode' = 'upsert',
+  'tikv.sink.transaction' = 'MINIBATCH',
+  'tikv.sink.delete_enable' = 'true'
+);
+
+INSERT INTO `tidb`.`test`.`target_table` SELECT id,name FROM `tidb`.`test`.`src_target`;
```

Comment:
Is this line necessary? Looks like it's executing batch inserting?

Reply:
it is necessary. 
first, as for like batch：batch or streaming depends on the config `tidb.streaming.source` in catalog. I don't known if is there any better way like `select from kafka topic`. I only find this way in the doc.
second, as for like insert：flink does not support delete statement. `flink-connector-jdbc` will also delete with insert  statement